ui/imports/shared/views/chat/InvitationBubbleView.qml

```diff
@@ -176,21 +176,35 @@ Control {
             color: Style.current.separator
         }
 
-        StatusFlatButton {
-            id: joinBtn
-
+        Item {
             Layout.fillWidth: true
             Layout.preferredHeight: 44
+            clip: true
+
+            StatusFlatButton {
+                id: joinBtn
+                width: parent.width
+                height: (parent.height+d.radius)
+                anchors.top: parent.top
+                anchors.topMargin: -d.radius
```

Comment:
tricky!

Reply:
Indeed but kinda standard in QML 😊

Reply:
The other way would be to use some OpacityMask but this involves a GL shader, while this one has `clip`ping. Actually dunno which one would be better/faster but hey, it works :)